Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -859,6 +859,11 @@ static String mutationObserverNotificationString()
     return m_scaleFactor;
 }
 
+CGFloat UnifiedPDFPlugin::contentScaleFactor() const
+{
+    return m_scaleFactor * m_documentLayout.scale();
```

Comment:
Looks like we have at least 3 uses of this product inside `UnifiedPDFPlugin.mm`. Would you like to replace those with `contentScaleFactor()` instead?

I see uses in `centeringOffset()`, `updateSnapOffsets()`, and `searchInDictionary()`.

Reply:
I think the logic in centeringOffset was slightly different in that it didn't allow just a simple swap in, but I was able to do the other 2